packages/card/src/Card.ts

```diff
@@ -266,77 +272,105 @@ export class Card extends LikeAnchor(
+        the role for the content wrapper can remain undefined. */
+        const roleForWrapper = this.role === 'row' ? 'gridcell' : undefined;
         return html`
-            ${this.renderImage()}
-            <div class="body">
-                <div class="header">
-                    ${this.renderHeading}
-                    ${this.variant === 'gallery'
-                        ? this.renderSubtitleAndDescription
-                        : nothing}
-                    ${this.variant !== 'quiet' || this.size !== 's'
+            <div class="wrapper" role=${ifDefined(roleForWrapper)}>
+                ${this.renderImage()}
+                <div class="body">
+                    <div class="header">
+                        ${this.renderHeading}
+                        ${this.variant === 'gallery'
+                            ? this.renderSubtitleAndDescription
+                            : nothing}
+                        ${this.variant !== 'quiet' || this.size !== 's'
+                            ? html`
+                                  <div
+                                      class="action-button"
+                                      @pointerdown=${this.stopPropagationOnHref}
+                                  >
+                                      <slot name="actions"></slot>
+                                  </div>
+                              `
+                            : nothing}
+                    </div>
+                    ${this.variant !== 'gallery'
                         ? html`
-                              <div
-                                  class="action-button"
-                                  @pointerdown=${this.stopPropagationOnHref}
-                              >
-                                  <slot name="actions"></slot>
+                              <div class="content">
+                                  ${this.renderSubtitleAndDescription}
                               </div>
                           `
                         : nothing}
                 </div>
-                ${this.variant !== 'gallery'
+                ${this.href
+                    ? this.renderAnchor({
+                          id: 'like-anchor',
+                          labelledby: 'heading',
+                      })
+                    : nothing}
+                ${this.variant === 'standard'
+                    ? html`
+                          <slot name="footer"></slot>
+                      `
+                    : nothing}
+                ${this.toggles
                     ? html`
-                          <div class="content">
-                              ${this.renderSubtitleAndDescription}
+                          <sp-popover
+                              class="checkbox-toggle"
+                              @pointerdown=${this.stopPropagationOnHref}
+                          >
+                              <sp-checkbox
```

Comment:
nit: I am not sure on the historical context of adding a direct element dependency of checkbox which reduces the flexibility and reusability. We should avoid this in future. Rather add slots and surface up extensibility should be a nicer way to move forward.